drake/doc/ubuntu.rst

```diff
@@ -35,8 +35,9 @@ Install the prerequisites::
 Download the external dependencies::
 
     cd drake-distro
-    make options  # use the GUI to choose which externals you want, then press 'c' to configure, then 'g' to generate makefiles and exit
-    make download-all
+    env CXX=g++-4.9 CC=gcc-4.9 make options
+    # Use the GUI to choose which externals you want, then press 'c' twice to configure, then 'g' to generate makefiles and exit.
```

Comment:
Here you need to press 'c' twice. I had someone in the past asking me why their configuration didn't go through. Maybe worth adding that?